OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs

```diff
@@ -55,13 +57,13 @@ private void WriteOpenMensaStandardInformation(XmlTextWriter xmlWriter)
             xmlWriter.WriteStartElement("canteen");
         }
 
-        private void WritePriceInformation(Weekday weekday, XmlTextWriter xmlWriter)
+        private void WritePriceInformation(Dish dish, XmlTextWriter xmlWriter)
         {
             foreach(string roleName in roleNames)
```

Comment:
- pleace change the foreach loop to a for loop to remove the unnecessary _priceCounter